apps/frontend/package.json

```diff
@@ -13,19 +14,28 @@
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-scripts": "5.0.1",
-    "typescript": "^4.7.3",
-    "tailwindcss": "3.1.2",
-    "@babel/core": "7.18.2"
+    "typescript": "^4.7.3"
   },
   "devDependencies": {
+    "@sovryn/eslint-config-custom": "*",
     "@sovryn/tsconfig": "*",
-    "@sovryn/eslint-config-custom": "*"
+    "autoprefixer": "^10.4.7",
+    "mini-css-extract-plugin": "^2.6.1",
+    "postcss": "^8.4.14",
+    "tailwindcss": "^3.1.3",
+    "typescript": "^4.7.4",
+    "postcss-normalize": "^10.0.1",
+    "postcss-import": "^14.1.0",
+    "postcss-nested": "^5.0.6",
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-preset-env": "^7.7.1",
+    "postcss-nesting": "^10.1.8",
```

Comment:
I think most of these could be moved to packages/tailwind lib.
And @sovryn/tailwind should be added as dependency here